### PR TITLE
Update WARNING message to clarifies the scenario

### PR DESCRIPTION
**Pull request by camilamacedo86**
Update WARNING message to clarifies that the unhealthy scenario is only faced after add new nodes.

[sc-57227]

### CODE REVIEW

File: pkg/cluster/purge_node.go [hunk 1]
```diff
@@ -100,14 +100,15 @@ func (c *Controller) PurgeNode(ctx context.Context, name string, rook bool, rook
...
 				"- Stop the Rook operator (kubectl -n rook-ceph scale --replicas=0 deployment.apps/rook-ceph-operator) \n"+
-				"- Delete the deployment for the failed mon (kubectl -n rook-ceph get pod -l app=rook-ceph-mon) \n"+
+				"- Delete the deployment for the failed mon (find out the Peding pod to be deleted with: kubectl -n rook-ceph get pod -l app=rook-ceph-mon) \n"+
 				"- Edit the configmap rook-ceph-mon-endpoints and (carefully) remove the failed mon from the list (kubect -n rook-ceph edit configmaps rook-ceph-mon-endpoints) \n"+
 				"- Start the Rook operator (kubectl -n rook-ceph scale --replicas=1 deployment.apps/rook-ceph-operator) \n "+
-				"- Ensure that Ceph came back in a healthy state (kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status) ", rookVersion)
+				"- Ensure that Ceph came back in a healthy state (kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status) \n"+
+				"- For further information see: https://community.replicated.com/t/managing-nodes-when-the-previous-rook-version-is-in-use-might-leave-ceph-in-an-unhealthy-state-where-mon-pods-are-not-rescheduled/1099", rookVersion)
```
Comment (camilamacedo86) [inline pkg/cluster/purge_node.go:111]:
```suggestion
				"For further information see: https://community.replicated.com/t/managing-nodes-when-the-previous-rook-version-is-in-use-might-leave-ceph-in-an-unhealthy-state-where-mon-pods-are-not-rescheduled/1099", rookVersion)
```

**Review by ricardomaraschini — CHANGES_REQUESTED** [review]
I have rephrased the whole thing, let me know what you think:

The version of Rook currently in use is %s and it is suggested to consider updating to the
 latest version for optimal performance. More information can be found at https://kurl.sh/docs/install-with-kurl/managing-nodes#rook-ceph-cluster-prerequisites. It's worth noting that using this version of Rook to manage nodes may result in an unhealthy Ceph cluster. If new nodes are added, it is recommended to check the status of Ceph (using the command "kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status"). If Ceph is found to be unhealthy, the following steps are suggested:

- Stop the Rook operator (by running "kubectl -n rook-ceph scale --replicas=0 deployment.apps/rook-ceph-operator")
- Delete the deployment for the failed mon (by identifying the pending pod to be deleted with "kubectl -n rook-ceph get pod -l app=rook-ceph-mon")
- Edit the configmap rook-ceph-mon-endpoints and remove the failed mon from the list (by running "kubectl -n rook-ceph edit configmaps rook-ceph-mon-endpoints")
- Start the Rook operator (by running "kubectl -n rook-ceph scale --replicas=1 deployment.apps/rook-ceph-operator")
- Ensure that Ceph has returned to a healthy state (by running "kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status"). For more information, please refer to https://community.replicated.com/t/managing-nodes-when-the-previous-rook-version-is-in-use-might-leave-ceph-in-an-unhealthy-state-where-mon-pods-are-not-rescheduled/1099.

**Comment by camilamacedo86** [comment]
Hi @ricardomaraschini,

After your comment I think we need to to be less verbose here and has no need to duplicated the info. So, could you please check it now.

**Comment by camilamacedo86** [comment]
Hi @ricardomaraschini,

Thank you. I think we can be less verbose here and do not duplicated the info. So, I applied your recommendations but removed the steps. WDYT?

**Pull request merged**